src/main/java/org/galatea/starter/ApiError.java

```diff
@@ -0,0 +1,21 @@
+package org.galatea.starter;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+
+import org.springframework.http.HttpStatus;
+
+@Getter
+@ToString
+@AllArgsConstructor
+public class ApiError {
```

Comment:
Do we need to have this class? It seems to be used only in the exception handler to communicate the HTTP response code to be returned to the private `buildResponseEntity()` method so in that case what is the downside of just having the `HttpStatus` and message passed in without the wrapper?